numl/NeutrinoML/HDF5Maker_module.cc

```diff
@@ -73,8 +75,11 @@ class HDF5Maker : public art::EDAnalyzer {
 
   string fTruthLabel;
   string fHitLabel;
+  string fHitTruthLabel;
   string fSPLabel;
 
+  bool fUseMap;
```

Comment:
If HItTruthLabel is specific to the uboone backtracking, can we remove UseMap as a FHICL parameter and just get the code to use the particle inventory service if HitTruthLabel is an empty string, and use uboone-style truth mapping if it's not empty?

Reply:
For now it's uboone-specific, but I am not sure what e.g. SBN experiments are (will be) doing. I thought it was cleaner with a bool, but the empty string is also a valid option. I don't have a strong preference.